aws-auth-cognito/build.gradle

```diff
@@ -30,7 +30,6 @@ dependencies {
     implementation dependency.androidx.security
     implementation dependency.androidx.browser
 
-    implementation dependency.aws.core
     implementation dependency.aws.cognitoidentity
     implementation dependency.aws.cognitoidentityprovider
```

Comment:
are these still Android SDK dependencies?

Reply:
No, we should have done better job of renaming them as I deleted them at first 🤦